app/views/facility_accounts_reconciliation/_action_row.html.haml

```diff
@@ -1,10 +1,19 @@
 .row.table-actions.form-horizontal
   .span2.select_all_none= select_all_link
+  .span1.pull-right= submit_tag t("facility_journals.show.submit"), class: ["btn", "btn-primary"], data: { disable_with: t("facility_journals.show.submit") }
```

Comment:
These changes are for external billing (credit cards and POs), journals are for internal payment sources.
```suggestion
  .span1.pull-right= submit_tag t("facility_accounts_reconciliation.index.submit"), class: ["btn", "btn-primary"], data: { disable_with: t("facility_accounts_reconciliation.index.submit") }
```